data/hai/hai reveal 1 intro.txt

```diff
@@ -47,6 +47,18 @@ mission "Hai Reveal: Reputation Reset"
 
 
 
+mission "Hai Reveal: Wanderers in Progress Reputation Fix"
```

Comment:
Why is there an apostrophe here?

This probably shouldn't require HR to have begun, someone could have started interactions with the Wanderers pre-FW and then updated and got HR and this wouldn't do anything for them.
It should instead require first contact with the Wanderers and *not* already have the event that lowers the required reputation (to avoid lowering it twice). Similarly, the latter calling of this event should also be changed to not occur if it has occured here.

Reply:
We need to fix this another way, blocking the Wanderer campaign by HR makes no sense and is not acceptable

Reply:
> this wouldn't do anything for them.

Yes it would.

> to avoid lowering it twice

It merely sets the required access reputation to 0 so it doesn't matter.

Reply:
> We need to fix this another way, blocking the Wanderer campaign by HR makes no sense and is not acceptable 

You will never convince me of this point, the explicit intent for it to do so was part of the original PR description from March 2020. The *current* start of the Wanderer campaign is bogus and terrible writing without something leading into it.

**Since people complain so much**, I've had some conversations with people and **worked out an alternative way to get into the Wanderer campaign** through a different entry point, but the *current* narrative way into Wanderers was explicitly *intended* to be blocked and you will not convince me that it should be otherwise.

Reply:
> > this wouldn't do anything for them.
> 
> Yes it would.

Can you explain?

Reply:
The first time they land literally anywhere the pre-conditions are set. This then checks as soon as the preconditions are set if you've already started Wanderers and then re-enables access for you.

Reply:
Which precondition? `Hai Secret Leaker Begins: failed`?
Here's that mission:
```
mission "Hai Secret Leaker Begins"
	landing
	invisible
	to offer
		has "main plot completed"
	on offer
		event "pug media coverage" 20 30
		fail
```
This will only be set after `main plot completed` is.
The case I am describing is exactly the one where a person starts interacting with the Wanderers before having `main plot completed` when they update.

Reply:
The alternative fix discussed on stream has been implemented.